### ISSUE TITLE
Missing space after paren removal from if statement starting with `!` operator

### ISSUE THREAD

**Issue by garricn** [nicklockwood/SwiftFormat#73]
Before:

```
if(!booleanStatement){
// code
}
```

Expected:

```
if !booleanStatement {
// code
}
```

Result:
```
if!booleanStatement {
// code
}
```

**Comment by nicklockwood** [comment]
@garricn can you confirm what version of SwiftFormat you are running? I believe this was fixed in version 0.19.

**Comment by garricn** [comment]
Oh no! I thought I updated to latest version. My bad, yes I was running 0.18 :( Anyhow, thank you!

**Comment by garricn** [comment]
Just confirmed update to 0.20 :)

**Comment by nicklockwood** [comment]
No problem! (btw I shipped 0.21 about 5 seconds ago :D)
